javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,143 @@
+//Flight booking fullname function
+function getFullname(firstname, surname, gender, useFormalName) {
+	if (gender === "male" && useFormalName === true) {
+		return "Your client full name is Lord " + firstname + " " + surname;
+	} else if (gender === "female" && useFormalName === true) {
+		return "Your client full name is Queen " + firstname + " " + surname;
+	} else if (gender === "" && useFormalName === true) {
+		return "Your client full name is " + firstname + " " + surname;
+	} else {
+		return "Please provide valid information";
+	}
+}
+console.log(getFullname("Joy", "Kumar", "male", true));
+console.log(getFullname("Riya", "Kumari", "female", true));
+console.log(getFullname("Oscar", "Roy", "", true));
+console.log(getFullname("", "", "", false));
+
+//End of Flight booking fullname function
+
+//Event application
+function getEventWeekday(days) {
+	const weekdays = [
+		"Sunday",
+		"Monday",
+		"Tuesday",
+		"Wednesday",
+		"Thursday",
+		"Friday",
+		"Saturday",
+	];
+	const todaysDate = new Date();
+	let today = todaysDate.getDay();
+	//console.log("Today is the " + weekdays[day]);
+
+	let eventDate = (today + days) % 7;
+	let eventweekday = weekdays[eventDate];
+	console.log(
+		`Today is ${weekdays[day]} and the event is in ${days} days. Therefore the event will be held on a ${eventweekday}.`
+	);
+}
+
+console.log(getEventWeekday(14));
+
+//End of Event application
+
+//Weather wear
+function getClothsBasedOnTempreture(tempreture) {
+	if (tempreture <= 10) {
+		return "winter jackets with thermal set";
+	} else if (tempreture > 10 && tempreture <= 25) {
```

Comment:
fine, also try switch case if it works for you.  check differences in medium.com

Reply:
yes sure.